Subsystems/nosSettingsSubsystem/Config/Types.fbs

```diff
@@ -0,0 +1,18 @@
+namespace nos.sys.settings;
+
+attribute "hidden_property";
+attribute "units";
+attribute "builtin";
+attribute "skip_make";
+attribute "skip_break";
+attribute "skip_delay";
+attribute "dynamic";
+attribute "resource";
+attribute "strict_type";
+
+table SettingsEntry{
+    name:string;
+    version:string;
+    type_name: string;
+    data: [ubyte] (dynamic: "type_name");
+}
```

Comment:
To support a module adding multiple settings with same type name (for example two float variables), the schema could be like this:

```
table SettingsEntry {
    key: string;
    type_name: string;
    data: [ubyte] (dynamic: "type_name");
}

table ModuleSettings {
   module_name: string;
   module_version: string;
   entries: [SettingsEntry];
}